bytes/bytes/raw/file_raw_repository.py

```diff
@@ -21,7 +21,6 @@ def create_raw_repository(settings: Settings) -> RawRepository:
 
 class FileRawRepository(RawRepository):
     UUID_INDEX = 3  # To reduce the number of subdirectories based on the uuid, see self._index()
-    _ENCODING = "utf-8"
```

Comment:
What is the purpose of this change?

Reply:
Discovered during looking into these issues that this is not used anywhere

Reply:
Raw files are also not guaranteed to be text, right?